errormessages/src/sass/main.scss

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+html, body {
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+}
+
+#messages {
+  list-style-type: none;
+  margin: 0 auto;
+  padding: 0;
+  width: 350px;
+}
+
+.message {
+  font-size: 90%;
+  color: white;
+  margin: 0 0 3px 0;
+  padding: 0;
+  height: 22px;
+  line-height: 22px;
+  text-align: center;
+  background-image: url('../images/alert-bg.png');
+  background-repeat: no-repeat;
+}
+
+/* Styles for ReactCSSTransitionGroup disabling for now till I can get them working
+.example-enter {
+	opacity: 0.01;
+  transition: opacity 1s ease-in-out;
+  -moz-transition: opacity 1s ease-in-out;
+  -webkit-transition: opacity 1s ease-in-out;
+}
+.example-enter.example-enter-active {
+	opacity: 1;
+}
+
+.example-leave {
+	opacity: 1;
+	transition: opacity 1s ease-in-out;
+  -moz-transition: opacity 1s ease-in-out;
+  -webkit-transition: opacity 1s ease-in-out;
+}
+.example-leave.example-leave-active {
+	opacity: 0.01;
+}
```

Comment:
Do not submit commented out code.  Create a branch to save this on if you want to save it and then delete it on the branch you're submitting for review.